### DISCUSSION TITLE
How can I change the URL's of every entry in a group/category?

### DISCUSSION THREAD

**Discussion by K4LCIFER** [FreshRSS/FreshRSS#6587] [Q&A] [answered]
I have a large amount of entries in one group/category which all come from the same domain. The domain has since changed (but the URL structure stays the same), so every entry must be updated. This would be very tedious to do manually, so I'm wondering how I could do this by editing the database. I would need to extract all the entries from that group, and be able to individually modify each entries URL. The procedure would be something like:
1. Retrieve first entry from group from the database
2. Extract the entry's URL
3. Through some external script, modify the URL to swap the old domain for the new one
4. Write the new URL to the entry
5. Repeat the process for all entries in the category.

I'm not sure how to interact with the database to accomplish what I want.

**Comment by Alkarex** [comment, marked as answer]
Hello,
At the moment, the main approach is likely to be to rewrite directly in database using SQL.
Connecting to your database depends on the type of database and setup you have chosen.
